server/main.go

```diff
@@ -59,7 +58,7 @@ func (app *App) setupRoutes() {
 }
 
 func (app *App) Start() {
-	app.server.Run(":" + strconv.Itoa(app.conf.Port))
+	app.server.Run("localhost:" + strconv.Itoa(app.conf.Port))
```

Comment:
did you encounter an error or something that required you to add the `localhost`?

Reply:
I believe Joseph added this line in. I forgot why, but it has something to do with the server and his Mac @joseph1wan

Reply:
I added this in so that it would stop prompting me to accept the connection every time I started `gin`.